### PR TITLE
Restructuring of folders

### PR DESCRIPTION
**Pull request by rodri-98**

### CODE REVIEW

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizer/TypeModelRecognizer.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.jalau.at18.searchobject.modelrecognizer;
+package org.jalau.at18.searchobject.model.objectrecognizer.recognizer;
 
 public enum TypeModelRecognizer {
     YOLO("yolo"),
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizer/TypeModelRecognizer.java:4]: You can have "yolo" as a constant.
Example:  `public static final String YOLO_STR = "YOLO";`

and you could also implement it for the lines 5 and 6.

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/ssd/utils/LabelUtils.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 
 import com.google.protobuf.TextFormat;
-import com.machinelearning.ssd.protos.StringIntLabelMapOuterClass;
 import java.nio.charset.StandardCharsets;
 
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.ssd.protos.StringIntLabelMapOuterClass;
+
 public class LabelUtils {
     public static String[] loadLabels() throws Exception {
         String text = new String(FileUtils.getBytes("labels/mscoco_label_map.pbtxt"), StandardCharsets.UTF_8);
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/ssd/utils/LabelUtils.java]: You can have this as a constant "labels/mscoco_label_map.pbtxt"

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package org.jalau.at18.searchobject.model.yolo.TransformTXT;
+package org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.yolo.TransformTXT;
 
 import java.io.*;
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java]: Use single imports

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
 import java.util.*;
```
Comment (FernandezRobert) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java]: use single imports

**Review by FernandezRobert — COMMENTED** [review]
Please fix minor fixes that were carried over.

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java (hunk 4 above)
Comment (rodri-98) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java]: It's already changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java (hunk 3 above)
Comment (rodri-98) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/yolo/TransformTXT/Transform.java]: It's already changed

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/ssd/utils/LabelUtils.java (hunk 2 above)
Comment (rodri-98) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizertypes/ssd/utils/LabelUtils.java]: It's already added a constant

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizer/TypeModelRecognizer.java (hunk 1 above)
Comment (rodri-98) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizer/TypeModelRecognizer.java:4]: It's a enum so it's not necessary to have "yolo" as a constant

**Review by FernandezRobert — APPROVED** [review]
Nice job.

**Review by DaniMB95 — APPROVED** [review]
nice job :)

**Review by ferAguilar02 — APPROVED** [review]

**Pull request merged**